scripts-dev/check_schema_delta.py

```diff
@@ -68,7 +74,10 @@ def main(force_colors: bool) -> None:
 
     seen_deltas = False
     bad_files = []
+    changed_files = []
     for diff in diffs:
+        changed_files.append(diff.b_path)
```

Comment:
This is currently above the `SCHEMA_FILE_REGEX` check. I assume we only want to work with delta files so we probably want to re-arrange a bit. And might as well have a better name for these things to make it clear

```suggestion
        changed_delta_files.append(diff.b_path)
```

Reply:
haha, :sob: